packages/suite/src/components/suite/Labeling/components/AddressLabeling.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as accountUtils from '@wallet-utils/accountUtils';
+import { useSelector } from '@suite-hooks';
+import { AccountLabeling } from './AccountLabeling';
+
+interface AddressLabelingProps {
+    address?: string | null;
+    knownOnly?: boolean;
```

Comment:
not sure about name of this prop, maybe `showKnownOnly` or smthing like that would be better

or maybe `shouldShowKnownOnly`, I know it's quite long but we should preserve is/has/should prefix for booleans

Reply:
done